.env_SAMPLE

```diff
@@ -51,11 +51,26 @@ export ALLOW_ADMIN_URL=True
 export DATABASE_URL=sqlite:///db.sqlite3
 #export DATABASE_URL=postgres://cfpb@localhost/cfgov
 
-# The use of default_storage_engine here assumes use of MySQL 5.7. For
-# MySQL 5.5 or earlier, use storage_engine instead.
+#DEPRECATED:
 #export DATABASE_URL=mysql://root@localhost/v1?init_command='SET default_storage_engine=MYISAM'
 
-# Required for existing MySQL-specific database scripts.
+#########################################################################
+# Postgres env vars to make tools like psql easier to work with
+#
+# See https://www.postgresql.org/docs/current/static/libpq-envars.html
+#########################################################################
+
+#export PGHOST=localhost
```

Comment:
Looking at these lines again, I wonder about the need/value of including them here. Would it be better to just encourage people to use `cfgov/manage.py dbshell`, and stick to the single `DATABASE_URL` variable? I'm not sure the increased complexity of more variables is worth being explicit.

Reply:
I'm all for reducing the number of environment variables here, so I'm glad to remove these in that case 👍